database/index.js

```diff
@@ -1,9 +1,10 @@
 const { Pool } = require('pg');
 
 const pool = new Pool({
-  user: process.env.PG_USER,
+  user: 'postgres',
   host: 'localhost',
   database: 'pacific',
+  password: 'admin',
```

Comment:
Maybe we can have user/password variables in the `.env` file? Otherwise we'd all need to have the same account.